specialists/agents/main.py

```
@@ -235,13 +235,13 @@ async def process_loneliness_companion(request: LonelinessCompanionRequest):
             checkpoint_status=result.get("checkpoint_status", "completed"),
             requires_human=result.get("requires_human", False),
             processing_time=processing_time,
-            agent_type="loneliness_companion",
+            agent_type="loneliness",
             agent_specific_data=result.get("agent_specific_data", {})
         )
         
     except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)